src/psm/src/ir_solver.cpp

```diff
@@ -332,6 +332,12 @@ void IRSolver::ReadC4Data() {
           "Y direction bump pitch is not specified, defaulting to {}um.",
           m_bump_pitch_default);
     }
+    if (m_node_density_um > 0) {
+      m_node_density = m_node_density_um * unit_micron;
```

Comment:
I don't see this being used anywhere.  What is its purpose?

Reply:
> I don't see this being used anywhere. What is its purpose?

It is used later in this same file. It is just not highlighted in the changes